### PR TITLE
Allow TLS connections

### LINKED ISSUE #14: Option to enable AMQPS (TLS)
Need this for our applications on Heroku.

### PR DESCRIPTION
**Pull request by jage**
Close #14

### CODE REVIEW

File: lib/twingly/amqp/connection.rb [hunk 1]
```diff
@@ -32,6 +33,10 @@ def ruby_env
         ENV.fetch("RUBY_ENV")
       end
 
+      def tls?
+        !!ENV.fetch("AMQP_TLS") { false }
```
Comment (roback) [inline lib/twingly/amqp/connection.rb]: Why not use `ENV.has_key?` instead :) http://ruby-doc.org/core-2.2.0/ENV.html#method-c-has_key-3F

That would get rid of `!!`

Comment (jage) [inline lib/twingly/amqp/connection.rb]: Ah, much better, thanks.

Comment (roback) [inline lib/twingly/amqp/connection.rb]: Hehe, why are there so many methods that does the same thing? :) 

`include?`, `key?`, `member?`, `has_key?`

**Comment by roback** [comment]
LGTM :)

**Pull request merged**